redisinsight/ui/src/pages/workbench/components/query/QueryWrapper.tsx

```diff
@@ -18,6 +18,7 @@ export interface Props {
   query: string
   activeMode: RunQueryMode
   resultsMode?: ResultsMode
+  queryProps?: any
```

Comment:
Is there any chance to type-hint this, or it's a legacy and is not possible at the moment